extensions/amp-next-page/1.0/service.js

```diff
@@ -855,23 +858,28 @@ export class NextPageService {
    * @private
    */
   getRemotePages_() {
-    if (!this.nextSrc_) {
+    if (this.isFetching_ || !this.nextSrc_) {
```

Comment:
Why not store the fetching promise and return it?

Reply:
+1